### DISCUSSION TITLE
Not really a problem, but wondered about a variable that is initialized but not used (partial gallery)

### DISCUSSION THREAD

**Discussion by darrenaddy** [Lednerb/bilberry-hugo-theme#406] [General]
Hello and thanks for providing this great theme. I like the way you code as it is usually pretty easy for me to figure out.
I was wondering about the variable `$imageGalleryClass` initialized on Line 4 of the partial gallery.html
I don't see where it is used. Was this for some future use, perhaps?

**Comment by igor-baiborodine** [comment]
Hi, 

There is no variable `$imageGalleryClass` initialized on line 4 in the partial [gallery.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/master/layouts/partials/content-type/gallery.html):

![image](https://user-images.githubusercontent.com/1027701/176543033-ce3198c4-7942-4fe8-9b4f-6d0e8dbf64a7.png)
